src/SystemLoader.cc

```diff
@@ -41,10 +41,8 @@ class ignition::gazebo::SystemLoaderPrivate
   public: explicit SystemLoaderPrivate() = default;
 
   //////////////////////////////////////////////////
-  public: bool InstantiateSystemPlugin(const std::string &_filename,
-              const std::string &_name,
-              const sdf::ElementPtr &/*_sdf*/,
-              ignition::plugin::PluginPtr &_plugin)
+  public: bool InstantiateSystemPlugin(const sdf::Plugin &_plugin,
+              ignition::plugin::PluginPtr &_ignPlugin)
```

Comment:
How about calling the SDF one `_sdfPlugin` and the pointer `_plugin`? Gotta stop adding more `ign` to the codebase